src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -39,18 +39,41 @@ String privacySettingButtonValue = (Boolean) request.getAttribute("isPrivate")?
     }
   </style>
 
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"> </script>
```

Comment:
Now that we're relying on jquery, how about we load it only once, perhaps in the index page? Once it's load there, it should be accessible everywhere.

Reply:
The drop downs I access using jquery aren't working when I move that line to the index page. I'm not sure what i'm doing wrong.